lib/devise/models/database_authenticatable.rb

```diff
@@ -7,6 +7,10 @@ module Models
     # Authenticatable Module, responsible for hashing the password and
     # validating the authenticity of a user while signing in.
     #
+    # This module overrides the `password=` method and skips setting the password
```

Comment:
It is correct to say it defines a `password=` but not that it skips setting the password since for what this gem is concerned, the table should not have a password column.

So I think this would be something I'd accept:

```
# This module defines a `password=` method. This method will hash the argument and store it in the
# `encrypted_password` column.

Reply:
Fair enough, changes made, I believe it's accurate now? sorry for the inaccuracy :)